### ISSUE TITLE
bou.ke/monkey license

### ISSUE THREAD

**Issue by lburgazzoli** [apache/camel-k#2067]
https://camel.zulipchat.com/#narrow/stream/257299-camel-k/topic/bouk.2Fmonkey

license states:
```
Copyright Bouke van der Bijl
I do not give anyone permissions to use this tool for any purpose. Don't use it.
```

**Comment by lburgazzoli** [comment]
@astefanutti can you have a look ?

**Comment by astefanutti** [comment]
@lburgazzoli yes I’ll work on it.

**Comment by akihikokuroda** [comment]
Should this be fixed in v1.2.x in addition to the latest?  If so, I can make a PR for the release-1.2.x branch.  Thanks!
